Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -105,6 +105,16 @@ public final class RootTabBarViewController: UITabBarController {
     profileNav.setViewControllers([profileVC, threadsVC, messageThreadVC], animated: true)
   }
 
+  public func switchToCreatorMessageThread(projectId: Param, messageThread: MessageThread) {
+    self.switchToDashboard(project: nil)
```

Comment:
@stephencelis, it has first to navigate to the dashboard `self.switchToDashboard(project: nil)` and then try to obtain the first viewcontroller in the visual stack and if it is a DashboardViewController (navigation to the dashboard was successful) then continue to the message thread